javascript/src/main/utils/aggregateUserMessageCount.js

```diff
@@ -1,20 +1,27 @@
 const aggregateUserMessageCount = (agg_messages, agg_slackUsers) => {
     const userMessageCounts = [];
-
+    const usersAdded = [];
     let i;
     for(i = 0; i<agg_slackUsers.length;i++){
         let count = 0;
         let j;
         for(j=0; j < agg_messages.length; j++) {
-            if (agg_messages[j].user_profile !== null && agg_messages[j].user_profile.real_name === agg_slackUsers[i].real_name){
+            if (agg_messages[j].user_profile !== null && agg_messages[j].user_profile.real_name === agg_slackUsers[i].profile.real_name){
```

Comment:
Can you explain why this .profile was added?

Reply:
from what i looked through, the profile.real_name attribute was more reliable (more often correct + present) which is likely because of whatever fixes the profile attribute makes in order to store the real name. Worst case, real_name and profile.real_name are the same, best case, real_name is incorrect/nickname and profile.real_name is correct from what i could see.